getting-started.md

```diff
@@ -48,43 +46,51 @@ Once it is up and running you can get a shell inside the container with:
 See [Managing containers](https://discourse.ubuntu.com/t/managing-containers/17763) for more details.
 
 ## Create an application
-**AMS** provides functionality to manage Android applications for you. To let **AMS** manage your application, you need the APK of the application and a `manifest.yaml` which looks like this in its simplest form:
+
+You can create applications through the web-based dashboard or on the command line with AMS.
+
+Both alternatives require you to provide the APK of the Android application, its name and its [instance type](https://discourse.ubuntu.com/t/instances-types-reference/17764).
```

Comment:
Technically, the APK is not needed. If not specified you are presented with the Android 10 launcher
In terms of tutorial however I'm not sure whether we should use an APK or not, depends on the level of simplicity we're aiming at